Composer/plugins/samples/assets/shared/scripts/provisionComposer.js

```diff
@@ -170,6 +171,23 @@ const unpackObject = output => {
   return unpacked;
 };
 
+const getTenantId = async (accessToken) => {
```

Comment:
Can we reuse this function somehow? I'm not familiar whether we have a common codebase if we do, it would be great to avoid duplication.

Reply:
> Can we reuse this function somehow? I'm not familiar whether we have a common codebase if we do, it would be great to avoid duplication.

This function is used only in this provision file. And currently the provisionComposer.js is a single file to maintain all provision details.